externals/llvm-external-projects/torch-mlir-dialects/test/Conversion/TcpToTosa/binary_ops.mlir

```diff
@@ -0,0 +1,7 @@
+// RUN: torch-mlir-dialects-opt <%s -convert-tcp-to-tosa | FileCheck %s
+
+func.func @test_add(%arg0 : tensor<10x20xf32>, %arg1 : tensor<10x20xf32>) -> tensor<10x20xf32> {
+  // CHECK: %{{.*}} = "tosa.add"(%{{.*}}, %{{.*}}) : (tensor<10x20xf32>, tensor<10x20xf32>) -> tensor<10x20xf32>
+  %0 = tcp.add %arg0, %arg1 : tensor<10x20xf32>, tensor<10x20xf32> -> tensor<10x20xf32>
+  return %0 : tensor<10x20xf32>
```

Comment:
include linalg lowerings too so we can exercise the dynamic shape design

Reply:
+1

Reply:
I'm working on a follow up PR with Linalg lowerings.

Reply:
The initial PR should be a "scale model" for what we want the system to grow into, so we should include the linalg lowerings from the beginning.

Reply:
This also includes wiring up the e2e tests.

Reply:
Or we can submit this with no ops (or just a dummy op (literally call it DummyOp)) to get the build system boilerplate and then we can evaluate new ops one by one.

Reply:
> The initial PR should be a "scale model" for what we want the system to grow into, so we should include the linalg lowerings from the beginning.

Okay, fair enough.

I have removed all the ops in this PR and added only a dummy op to get the boilerplate code in first.